piptools/repositories/pypi.py

```diff
@@ -254,7 +254,7 @@ def get_dependencies(self, ireq):
                 # If a download_dir is passed, pip will  unnecessarely
                 # archive the entire source directory
                 download_dir = None
-            elif ireq.link and not ireq.link.is_artifact:
+            elif ireq.link and is_vcs_url(ireq.link):
```

Comment:
```suggestion
            elif ireq.link and ireq.link.is_vcs:
```
As I understand https://github.com/pypa/pip/commit/aea7d86a2271aa15779ab52cf495952ab381028e and https://github.com/pypa/pip/blob/76ae377d7e38b5df8587d63b52464a08471eb1d1/news/6883.trivial `is_artifact` should be replaced with `not_is_vcs`, right?

Reply:
Yes, it should be. `Link.is_vcs` property has appeared recently and is not compatible with `pip<19.3`, that's why it's [is_vcs_url(link)](https://github.com/jazzband/pip-tools/blob/73619fb3b8bd54974c5587bd658dc1ee9a317a50/piptools/_compat/pip_compat.py#L67-L72).

Reply:
I see, thanks for clarifying.